### ISSUE TITLE
Responsive navigation menu missing aria-expanded state and the invisible menu items are incorrectly focusable when it is collapsed.

### ISSUE THREAD

**Issue by JohnHewittCriterion** [nciinc/fs-permit-platform#499]
The responsive navigation menu that appears when the text is larger sized or the screen is small is missing a collapsed state to tell the screen reader that it is a dynamic control that opens and closes. 
 
Use aria-expanded=false when it's closed. 
 
The X close button should be more specific and say "Close Navigation Menu" not just "Close".
 
The invisible menu items are still focusable when they are in the hidden collapsed state even though you can't see the focus when tabbing to them. 
Hide the collapsed menu items using CSS display:none or visibility:hidden so they can't get focus. 

Linkage: CTF-39

**Comment by sadlerw** [comment]
@JohnHewittCriterion I think we changed the behavior of the sidebar navigation so that the sub-elements are removed completely from the dom when the navigation section is closed.  An older version of the code might have left them somehow, but we aren't seeing them there now.  Can you have someone confirm please? Thanks, Will

**Comment by JohnHewittCriterion** [comment]
We'll take a look at the nav this afternoon, thanks for the heads up
